pkg/packagekit/kpatch.jsx

```diff
@@ -0,0 +1,325 @@
+            action = this.handleInstall;
+        } else if (this.state.auto === false) {
+            state = _("Disabled");
+            actionText = _("Enable");
+        }
+
+        const details = [];
+        if (this.state.enabled !== this.state.futureEnabled) {
+            if (this.state.futureEnabled)
+                details.push(_("kpatch.service will be enabled. After each reboot patches will be applied."));
+            else
+                details.push(_("kpatch.service will be disabled. After reboot no patches will be applied."));
+        }
+        // FIXME - should we uninstall current patches as well?
+        if (this.state.auto !== this.state.futureAuto) {
+            if (this.state.futureAuto)
+                details.push(_("Current kernel patches will be installed. Also when a new kernel is available, patches will be automatically installed as well."));
+            else
+                details.push(_("For future kernel versions patches won't be installed."));
+        }
+
+        const body = <Stack hasGutter>
+            <StackItem>
+                <Switch isChecked={this.state.futureEnabled}
+                            label={_("Kernel live patching service is enabled")}
+                            labelOff={_("Kernel live patching service is disabled")}
+                            onChange={c => this.setState({ futureEnabled: c })} />
+            </StackItem>
+            <StackItem>
+                <Switch isChecked={this.state.futureAuto}
+                            label={_("Subscribed to available kernel patches")}
+                            labelOff={_("Not subscribed to kernel patches")}
+                            onChange={c => this.setState({ futureAuto: c })} />
+            </StackItem>
+            <StackItem>
+                <List>{details.map(d => <ListItem key={d}>{d}</ListItem>)}</List>
+            </StackItem>
+        </Stack>;
+
+        return (<>
+            <div id="kpatch-settings">
+                <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                    <Flex grow={{ default: 'grow' }} alignItems={{ default: 'alignItemsBaseline' }}>
+                        <FlexItem>
+                            <b>{_("Kernel live patching")}</b>
+                        </FlexItem>
+                        <FlexItem>
+                            {state}
+                        </FlexItem>
+                    </Flex>
+                    <Flex>
+                        <Button variant="secondary"
+                                isSmall
+                                disabled={!this.props.privileged}
+                                onClick={action}>
+                            {actionText}
+                        </Button>
+                    </Flex>
+                </Flex>
+            </div>
+            <Modal position="top" variant="small" id="kpatch-setup" isOpen={this.state.showModal}
+                title={_("Kernel live patching settings")}
```

Comment:
```suggestion
                title={_("Kernel live patch settings")}
```

```suggestion
                title={_("Kernel patch settings")}
```

What are we calling this? Kernel patches, kernel live patches, or what? We should be consistent.

Reply:
Ok, lets drop "live" from everywhere.